### ISSUE TITLE
🚨Add to Package List

### ISSUE THREAD

**Issue by Blackjacx** [Blackjacx/SHSearchBar#55]
After #54  is merged add this to the [package index](https://github.com/SwiftPackageIndex/PackageList#how-do-you-add-a-package)

**Comment by Blackjacx** [comment]
Added new PR: https://github.com/SwiftPackageIndex/PackageList/pull/639. 

View SHSearchbar @PackageIndex at https://swiftpackageindex.com/Blackjacx/SHSearchBar
